gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -0,0 +1,118 @@
+module OpenstackHandle
+  module HandledList
+    def handled_list(collection_type, options = {})
```

Comment:
I'd go for something boringly descriptive (and featuring a verb), like `expand_paginated_list`.

I'm not terribly offended by the idea of just naming the module after the only public method it provides.